### PR TITLE
Fixes for dark themes

### PR DESCRIPTION
**Pull request by msfjarvis**
* Fix some more warnings too
* Also delete tests

### CODE REVIEW

**Comment by boswelja** [comment]
tbh I was horrified at the state of substratum after I applied a dark theme to it.. Black text on a grey background etc
Some elements were still white too

**Comment by msfjarvis** [comment]
Didn't see anything of the sort with Domination, but that was a week ago. Lots of stuff changed, derps might have crept in.

File: app/src/main/res/layout/main_activity.xml [hunk 1]
```diff
@@ -43,6 +43,7 @@
 
         <FrameLayout
             android:id="@+id/main"
+            android:background="@color/splashscreen_background"
```
Comment (nicholaschum) [inline app/src/main/res/layout/main_activity.xml:46]: Why is the background of MainActivity being set by splashscreen background? This does not expose it but cause more issues.

**Comment by ivaniskandar** [comment]
I think it will be better if you split up the commit into individual commits that have it own purpose. Otherwise it still need some testing with the text exposure

**Comment by boswelja** [comment]
@iskandar1023 The text that I have set a colour value seems to look fine on your OrangeKitten theme in both light and dark themes

**Comment by nicholaschum** [comment]
I hope you aren't testing adapting Substratum to a theme that has not been updated for API25, as there have been style changes required by the themer to update.

**Comment by ivaniskandar** [comment]
hmm yes i hope so, because i haven't update mine.

**Comment by MSF-Jarvis** [comment]
@iskandar1023 What timing!

**Comment by boswelja** [comment]
@iskandar1023 is OrangeKitten not updated for api25?

**Comment by boswelja** [comment]
screenshot_20170115-182433
screenshot_20170115-182458
screenshot_20170115-182506

**Comment by ivaniskandar** [comment]
@boswelja not yet

**Comment by boswelja** [comment]
@iskandar1023 even tho theme card says nought - pixel n? rip

**Comment by MSF-Jarvis** [comment]
@boswelja I am insane.

**Comment by ivaniskandar** [comment]
Yes, my fault.

**Comment by boswelja** [comment]
welp, now i need a new theme to test this

**Comment by ivaniskandar** [comment]
device-2017-01-15-122838

**Comment by boswelja** [comment]
@iskandar1023 ooh, is that a test build?

**Comment by ivaniskandar** [comment]
Yes, notice the different color between the first and second card. The first have the exposure from this PR. Aaaaand AFAIK the current one without exposure is better.
*IMO

**Comment by boswelja** [comment]
I see, I knew something had to be wrong with a theme engine not being themed properly

**Comment by nicholaschum** [comment]
Break apart the PR with your Java 8 adaptations and that shall go in. API25 is a themer related issue and should be resolved with the themer. Sorry, it doesn't sound logical to have the theme app have further updates based on what themes support, such as recycling a splashscreen background for the main activity.

**Pull request closed without merging**